test/TextTableParser.Tests.ps1

```diff
@@ -0,0 +1,199 @@
+                    @{ ROW = 2;  "Property_01" = "-rw-r--r--"; "Property_02" = "1"; "Property_03" = "14512"; "Property_04" = "Oct"; "Property_05" = "12"; "Property_06" = "16:58"; "Property_07" = "NativeTableParser.pdb" }
+        },
+        @{
+            FileName = "ps.01.txt"
+            convertArgs = @{ }
+            Rows = 6
+            Results = @{ ROW = 0;  "PID" = "2596"; "TTY" = "ttys000"; "TIME" = "1:59.45"; "CMD" = "/usr/local/bin/pwsh -l" },
+                    @{ ROW = 1;  "PID" = "2601"; "TTY" = "ttys001"; "TIME" = "0:44.13"; "CMD" = "/usr/local/bin/pwsh -l" },
+                    @{ ROW = 2;  "PID" = "2661"; "TTY" = "ttys002"; "TIME" = "0:23.53"; "CMD" = "/usr/local/bin/pwsh" }
+        },
+        @{
+            FileName = "ps.02.txt"
+            convertArgs = @{ }
+            Rows = 88
+            Results = @{ ROW = 82;  "UID" = "0"; "PID" = "69835"; "PPID" = "2596"; "C" = "0"; "STIME" = "8:25AM"; "TTY" = "ttys000"; "TIME" = "0:00.01"; "CMD" = "/bin/ps -ef" },
+                    @{ ROW = 85;  "UID" = "501"; "PID" = "2669"; "PPID" = "2658"; "C" = "0"; "STIME" = "9:45AM"; "TTY" = "ttys003"; "TIME" = "0:23.04"; "CMD" = "/usr/local/bin/pwsh" },
+                    @{ ROW = 87;  "UID" = "501"; "PID" = "2676"; "PPID" = "2658"; "C" = "0"; "STIME" = "9:45AM"; "TTY" = "ttys005"; "TIME" = "0:23.16"; "CMD" = "/usr/local/bin/pwsh" }
+        },
+        @{
+            FileName = "ps.03.txt"
+            convertArgs = @{ }
+            Rows = 6
+            Results = @{ ROW = 0;  "UID" = "501"; "PID" = "2596"; "PPID" = "2587"; "F" = "4006"; "CPU" = "0"; "PRI" = "33"; "NI" = "0"; "SZ" = "39662748"; "RSS" = "167544"; "WCHAN" = "-"; "S" = "S`u{002B}"; "ADDR" = "0"; "TTY" = "ttys000"; "TIME" = "2:01.35"; "CMD" = "/usr/local/bin/pwsh -l" },
+                    @{ ROW = 5;  "UID" = "501"; "PID" = "2676"; "PPID" = "2658"; "F" = "4006"; "CPU" = "0"; "PRI" = "33"; "NI" = "0"; "SZ" = "39658344"; "RSS" = "136596"; "WCHAN" = "-"; "S" = "Ss`u{002B}"; "ADDR" = "0"; "TTY" = "ttys005"; "TIME" = "0:23.17"; "CMD" = "/usr/local/bin/pwsh" }
+                
+        },
+        @{
+            FileName = "ps.04.txt"
+            convertArgs = @{ }
+            Rows = 785
+            Results = @{ ROW = 782;  "PID" = "2676"; "TTY" = "ttys005"; "TIME" = "0:27.57"; "CMD" = "pwsh" },
+                    @{ ROW = 783;  "PID" = "94197"; "TTY" = "ttys006"; "TIME" = "1:07.57"; "CMD" = "pwsh" },
+                    @{ ROW = 784;  "PID" = "94200"; "TTY" = "ttys007"; "TIME" = "0:05.13"; "CMD" = "pwsh" }
+        },
+        @{
+            FileName = "tasklist.01.txt"
+            convertArgs = @{}
+            Rows = 46
+            Results = @{ ROW = 0;  "Image_Name" = "========================="; "PID" = "========"; "Session_Name" = "================"; "Session#" = "==========="; "Mem_Usage" = "============" },
```

Comment:
The conversion process should check for header symbols and remove them from parsed output. For example, if all the properties of a row contain the same character (`=`, `-`, `_`, etc) it should be removed.

Reply:
it's the etc I'm worried about. Right now, I'm not spending any time determining the _content_ of the data, just the column delineation. There might be a performance penalty if I have to carefully inspect the data. Also, I need those separators as part of the determination of what a column, so I can't preprocess it away.

Reply:
I've opened #34 to track